_docs/templates/email-template.md

```diff
@@ -21,32 +26,56 @@ if more information is needed links to control panels are provided.
 - Notifications and Tickets to customers
 - All message types and severity levels
 
-## When to use
+### When to use
 
 These email templates are for external communications to customers related to
 control panels when sending out notifications and tickets.
 
-## Quick View Template and Variable Guide
+{% endcolumn %}
+</div>
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Quick view template and variable guide
 
 {% figure [caption:"Quick guide for template and variable options"] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/EmailTemplate.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/EmailTemplate.svg)
 {% endfigure %}
 
+{% endcolumn %}
+
+</div>
+
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
 ## Typography and image guides
 
 {% figure [caption:"Quick guide for typography and image specifications"] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Email%20Detail%20-%20Spec.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Email%20Detail%20-%20Spec.svg)
```

Comment:
This one looks a little odd in places, too.
<img width="111" alt="screen shot 2017-11-20 at 11 00 08 am" src="https://user-images.githubusercontent.com/1538692/33030845-0639f324-cde2-11e7-8382-359c0810ef8d.png">
<img width="80" alt="screen shot 2017-11-20 at 11 00 37 am" src="https://user-images.githubusercontent.com/1538692/33030855-0fb6e060-cde2-11e7-8698-a6e6de321bc3.png">

Reply:
Need to remake this image. This is the current image that is on the live site as well. The difference between live and staging was the entire layout was borked on staging and good on live. Both used the same images however, so any issues with the images will exist in this updated branch and live.